### ISSUE TITLE
4B4F07D3 - Dynasty Warriors 5 Empires

### ISSUE THREAD

**Issue by michaelx333** [xenia-project/game-compatibility#1793]
[Marketplace](https://marketplace.xbox.com/en-us/product/dw5-empires/66acd000-77fe-1000-9115-d8024b4f07d3?nosplash=1)

Tested on https://github.com/xenia-project/xenia/commit/ddee85f0be00b3311a258a5ac3faee5e32b91de2

# Issues:
Freezes after starting a battle and the first character voice line finishes playing. Error shown in the screenshot.
Also has some graphic issues I think but it freezes before I can tell.

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/6754929/xenia.zip)

# Screenshot(s):
![Capture](https://user-images.githubusercontent.com/13000211/124284375-39959080-db4d-11eb-8e13-919ea0348ed1.PNG)

# Labels:
state-menus

**Comment by mitch03** [comment]
hey, any updates on this game working? thankyou!

**Comment by mitch03** [comment]
any news on this issue being resolved?

**Comment by Margen67** [comment]
> any news on this issue being resolved?

Here's the news: Read the README and only post compatibility reports. Thanks. https://github.com/xenia-project/game-compatibility#reportissue-guidelines

**Comment by mitch03** [comment]
> > any news on this issue being resolved?
> 
> Here's the news: Read the README and only post compatibility reports. Thanks. https://github.com/xenia-project/game-compatibility#reportissue-guidelines

Sorry I didn't know, just wanted to know if there was any progress as i love this game lol, sorry anyway!

**Comment by Pannn44** [comment]
the problem is still occurs

**Comment by michaelx333** [comment]
> the problem is still occurs

Yeah it's a shame. This is the only version of the game with upgraded graphics and nobody seems to care about fixing it, unfortunately. Also, this same issue is on all of the old Warriors games by Koei (Warriors Orochi/Samurai Warriors)... So it would fix all of them if it was fixed on just 1 of them!

**Comment by mitch03** [comment]
> > the problem is still occurs
> 
> Yeah it's a shame. This is the only version of the game with upgraded graphics and nobody seems to care about fixing it, unfortunately. Also, this same issue is on all of the old Warriors games by Koei (Warriors Orochi/Samurai Warriors)... So it would fix all of them if it was fixed on just 1 of them!

yea i hope the problem gets resolved

**Comment by NomadJack** [comment]
![image](https://github.com/user-attachments/assets/6e6236f9-3253-4677-b1e6-922c0cfbe935)

Crashes within seconds of GAMEPLAY starting :(

Seems to crash regardless of values used in the settings below;
**clear_memory_page_state =
use_dedicated_xma_thread = 
apu_max_queued_frames =**

Same with Samurai Warriors 2 and Samurai Warriors 2 Empires. (Same Engine)

**Comment by Margen67** [comment]
@NomadJack Xenia Canary discussion belongs here: https://github.com/xenia-canary/game-compatibility/
